dipu/torch_dipu/csrc_dipu/vendor/droplet/deviceimpl.cpp

```diff
@@ -14,7 +14,8 @@ using DROPLET_deviceId = int;
 // =====================
 
 void initializVendor() {
-
+  int num = -1;
```

Comment:
add comments to explain why needed add this get device count during initialization

Reply:
done